cmd/blockchaincmd/convert.go

```diff
@@ -403,6 +410,7 @@ func InitializeValidatorManager(
 			aggregatorAllowPrivatePeers,
 			aggregatorLogger,
 			validatorManagerAddrStr,
+			useACP99,
```

Comment:
With ACP 99, we no longer have to differentiate initializing validator manager for poa and pos. We can just initialize validator manager and if user wants to do pos, we can do additional steps after.

Example implementation:
https://github.com/ava-labs/avalanche-cli/blob/f0a6d96b7c4a29e1e5a9f81482450f92ba37d9bb/cmd/blockchaincmd/convert.go#L374

Reply:
the PT still supports v1.0.0